readme.md

```diff
@@ -47,13 +47,15 @@ const {app, BrowserWindow, ipcMain} = require('electron');
 const {download} = require('electron-dl');
 
 ipcMain.on('download-btn', (e, args) => {
-	download(BrowserWindow.getFocusedWindow(), args.url);
+	download(BrowserWindow.getFocusedWindow(), args.url)
+		.then(dl => console.log(dl.getSavePath()))
+		.catch(err => console.error(err));
```

Comment:
TIL this was legal in Node, and in Chrome since April :smile: I guess I had the more verbose form beaten into me by the old behavior in Chrome and never bothered to check.